module-4/backend/week_one.md

```diff
@@ -4,17 +4,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What's the most useful thing you learned from completing the intermission week work?
-2. What are some tools to help debug JavaScript code?
-3. What are some tools you need in order to unit test your JavaScript?
-4. What is the syntax for invoking a function?
-5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
-7. How do we setup a route when creating an API with Node and Express?
-8. What do we use Knex for?
-9. What's `npm` and what do we use it for?
+1. What's the most useful thing you learned from completing the intermission week work?  
+
+   Becoming comfortable with basic JavaScript.  
+   
+   
+2. What are some tools to help debug JavaScript code?  
+
+   debugger, chrome Dev Tools, pryJs, and my old favorite: console.log, seriously though, after Friday's lesson I've primarily used Chrome Dev Tools   
+   
+   
+3. What are some tools you need in order to unit test your JavaScript?  
+
+   mocha, chai  
+   
+   
+4. What is the syntax for invoking a function?  
+
+   functionName()  
+   
+   
+5. What's the difference between `==` and `===` in JavaScript?  
+
+   `==` is more strict than `===`  
+   
+   
+6. What's the difference between asynchronous and synchronous JavaScript?  
+
+   synchronous runs in order, i.e.,during the page load, asynchronous can start running in order but will finish whenever it's task is done and can be run at any time  
+   
+   
+7. How do we setup a route when creating an API with Node and Express?  
+
+   I don't know  
+   
+   
+8. What do we use Knex for?  
+
+   to interface between a node backend and a SQL database  
+   
+   
+9. What's `npm` and what do we use it for?  
+
+   `npm` is a package manager for JS.  We use it to manage dependencies in our projects, incorporate libraries.  Like `bundler` for Ruby.  
 
 #### Review  
-10. What's the MVC design pattern? Describe each part of MVC?
-11. What is AJAX? What are some benefits of using AJAX?
-12. What's a background worker? When would we want to use a background worker?
+10. What's the MVC design pattern? Describe each part of MVC?  
+   `MVC` - Model, View, Controller.  Model - manages db interactions, View - client side, Controller - directs traffic
+   
+11. What is AJAX? What are some benefits of using AJAX?  
+   `AJAX` - Asynchronous JS and XML(or JSON) - It can be run anytime and used to update elements on the page  
```

Comment:
This should have been a different question as well but some benefits of AJAX is to be able to make requests to a server.